pager: Use less instead of pager

pager is a Debianism. Istead use directly pager.

Upstream-Status: Inappropriate [OE-Core integration specific]

Suggested-by: Burton, Ross <ross.burton@intel.com>
Signed-off-by: Ricardo Ribalda <ricardo@ribalda.com>
diff --git a/lib/dpkg/dpkg.h b/lib/dpkg/dpkg.h
index 2bb067a..6cbce80 100644
--- a/lib/dpkg/dpkg.h
+++ b/lib/dpkg/dpkg.h
@@ -95,7 +95,7 @@ DPKG_BEGIN_DECLS
 #define MAXUPDATES         250
 
 #define DEFAULTSHELL        "sh"
-#define DEFAULTPAGER        "pager"
+#define DEFAULTPAGER        "less"
 
 #define MD5HASHLEN           32
 #define MAXTRIGDIRECTIVE     256
